git.lua

```diff
@@ -112,6 +112,30 @@ local function branches(token)
     return res
 end
 
+local function alias(token)
```

Comment:
See this

http://stackoverflow.com/a/22875140/848072

or my dotfiles

https://github.com/albfan/dotfiles/blob/master/gitconfig#L3

or related doc

https://git.wiki.kernel.org/index.php/Aliases#Aliases

alias are contractions to ease execution of regular basis commands